training/backends/kubernetes.js

```diff
@@ -147,12 +168,12 @@ module.exports = async function execTask(job, spec) {
                     containers: [
                         {
                             name: 'main',
-                            image: Config.TRAINING_KUBERNETES_IMAGE + (spec.gpu > 0 ? '-cuda' : ''),
+                            image: Config.TRAINING_KUBERNETES_IMAGE + (spec.requests.gpu > 0 ? '-cuda' : ''),
                             imagePullPolicy: 'Always',
                             args: [
                                 'run-training-task',
                                 '--task-name', spec.name,
-                                '--job-id', job.id,
+                                '--job-id', `${job.id}`,
```

Comment:
Our code style convention is to use `String(foo)` to force something into a String.